integ-test/src/test/java/org/opensearch/sql/sql/DateTimeFormatsIT.java

```diff
@@ -29,6 +30,7 @@ public class DateTimeFormatsIT extends SQLIntegTestCase {
   public void init() throws Exception {
     super.init();
     loadIndex(Index.DATE_FORMATS);
+    loadIndex(Index.DATA_TYPE_NONNUMERIC);
```

Comment:
nit: curious why you added the date_nanos_value to DATA_TYPE_NONNUMERIC rather than DATE_FORMATS

Reply:
I have it in both